lib/inaturalist_map_server.js

```diff
@@ -60,6 +60,11 @@ InaturalistMapserver.prepareQuery = function( req, callback ) {
   req.elastic_query.fields = InaturalistAPI.defaultMapFields( );
   // requiring mappable=true to be included in map tiles
   req.query.mappable = "true";
+  if ( req.query.captive === "any" ) {
```

Comment:
do we need this code around "any" here? I think this conditional just needs to be:
```
  if ( req.query.captive !== "true" ) {
    req.query.captive = "false";
  }
```
and then we also wouldn't need the test about captive=any doesn't need to stick around